backend-go/internal/core/users/handler.go

```
@@ -1,6 +1,9 @@
 package users
 
 import (
+	"errors"
```

golangci-lint (1.64.8)

File is not properly formatted

(goimports)